lib/_debugger.js

```diff
@@ -539,14 +529,10 @@ Client.prototype.mirrorObject = function(handle, depth, cb) {
 
 
       var keyValues = [];
-      handle.properties.forEach(function(prop, i) {
+      handle.properties.forEach((prop, i) => {
         var value = res[prop.ref];
-        var mirrorValue;
-        if (value) {
-          mirrorValue = value.value ? value.value : value.text;
-        } else {
-          mirrorValue = '[?]';
-        }
+        var mirrorValue = value ?
```

Comment:
Can you make the variable `const`. Also, `(value.value ? value.value : value.text)` can probably just be `value.value || value.text`